server/src/controllers/__tests__/map.spec.ts

```diff
@@ -33,7 +33,10 @@ jest.mock("../../db/index", () => {
 describe("MapController", () => {
   describe("getMap", () => {
     it("should get a map", async () => {
-      const mockMap = { id: 1, day: new Date(), par: 3, hole_x: 1, hole_y: 1, ball_x: 1, ball_y: 1 }
+      const date = new Date()
+      date.setUTCHours(0, 0, 0, 0)
+      console.log(date)
```

Comment:
make sure hours, minutes, seconds, and ms are 0 because all we care about is the date